src/main/kotlin/no/nav/syfo/application/ApplicationEnvironment.kt

```diff
@@ -3,10 +3,12 @@ package no.nav.syfo.application
 data class Environment(
     val applicationName: String = "syfooversiktsrv",
 
-    val azureAppClientId: String = getEnvVar("AZURE_APP_CLIENT_ID"),
-    val azureAppClientSecret: String = getEnvVar("AZURE_APP_CLIENT_SECRET"),
-    val azureAppWellKnownUrl: String = getEnvVar("AZURE_APP_WELL_KNOWN_URL"),
-    val azureTokenEndpoint: String = getEnvVar("AZURE_OPENID_CONFIG_TOKEN_ENDPOINT"),
+    val azure: ApplicationEnvironmentAzure = ApplicationEnvironmentAzure(
+        appClientId = getEnvVar("AZURE_APP_CLIENT_ID"),
+        appClientSecret = getEnvVar("AZURE_APP_CLIENT_SECRET"),
+        appWellKnownUrl = getEnvVar("AZURE_APP_WELL_KNOWN_URL"),
+        tokenEndpoint = getEnvVar("AZURE_OPENID_CONFIG_TOKEN_ENDPOINT"),
+    ),
```

Comment:
Nå er det enklere å lese! 🏆

Reply:
Ja, jeg synes det. Jobber med  variabler for redis og client og nå. Blir mindre kode av det og 👍